Userland/Libraries/LibVideo/VP9/TreeParser.cpp

```diff
@@ -665,80 +665,86 @@ u8 TreeParser::calculate_token_probability(u8 node)
 
 void TreeParser::count_syntax_element(SyntaxElementType type, int value)
 {
+#define INCREMENT_ELEMENT(element)                                    \
```

Comment:
Instead of adding a macro for this can't we just use `AK::clamp`?

Reply:
I changed it to use a local lambda to set `x = min(x + 1, 255)` for each element instead, and reworded the commit message since clamp is invalid for unsigned integers.